wordpress-mobile/WordPress-Android
# Update jetpack.json for login animations

Fixes #6960. /cc @theck13 @hypest 

@elibud It turns out we changed those screens that used Jetpack logo, so aside from this login animation we don't use the Jetpack logo as also mentioned by @theck13.

To test:
* Check the Jetpack logo in the login screen's last page animation is the correct version

Old one:
![screenshot_1513031495](https://user-images.githubusercontent.com/662023/33857641-7b45645a-de9a-11e7-981e-215d2ff492ac.png)

New one:
![screenshot_1513031559](https://user-images.githubusercontent.com/662023/33857644-7e735542-de9a-11e7-8f94-431dda664778.png)
